backend/package.json

```diff
@@ -35,14 +39,29 @@
     "@babel/node": "^7.8.7",
     "@babel/preset-env": "^7.9.6",
     "@types/jest": "^25.2.1",
+    "babel-eslint": "^10.1.0",
     "babel-jest": "^26.0.1",
-    "eslint": "^7.0.0",
+    "eslint-config-airbnb-base": "^14.2.0",
+    "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.20.1",
-    "eslint-plugin-jest": "^23.10.0",
+    "eslint-plugin-jest": "^23.20.0",
     "eslint-restricted-globals": "^0.2.0",
+    "husky": "^4.2.5",
     "jest": "^26.0.1",
+    "lint-staged": "^10.2.11",
     "nodemon": "^2.0.2",
     "prettier": "2.0.5",
     "supertest": "^4.0.2"
+  },
+  "lint-staged": {
+    "*.{js,jsx, sass, json, css}": [
+      "eslint --fix",
+      "prettier --write"
```

Comment:
@marcianszlovak dont fix it when its in the precommit hook, there can be a script for that, but its not good to do it automatically for every commit